packages/opencensus-exporter-stackdriver/src/stackdriver-stats-utils.ts

```diff
@@ -237,6 +287,37 @@ function leftZeroPad(ns: number) {
   return `${pad}${str}`;
 }
 
+function getGcpResourceLabelsMappings() {
```

Comment:
What's the motivation for having these be functions and not just assign the values to the constants above?

IMO, having the constant right next to its value would make it clearer and more compact, but up to you on this.